source/index.html.md

```diff
@@ -444,27 +423,54 @@ window.drawVisualization = function(p) {
 ...
 ```
 
-The last remaining task for our application is displaying the resource information we've retrieved. In "index.html" we define a table with several id place holders. On a success from ```extractData``` we'll call ```drawVisualization``` which will show the table div as well as filling out the relevant sections.
+The last remaining task for our application is displaying the resource information we've retrieved. In `index.html` we define a table with several id place holders. On a success from ```extractData``` we'll call ```drawVisualization``` which will show the table div as well as filling out the relevant sections.
 
 # Test your App
-> To re-deploy the GitHub Pages site commit your changes and run:
+> To re-deploy the GitHub Pages site, commit your changes and make sure your gh-pages branch is up to date.
 
-```bash
-$ ./deploy.sh
-```
+Now that we have a snazzy SMART app, it's time to test it.
+
+Next log back into the <a href="https://code.cerner.com/developer/smart-on-fhir/apps" target="_blank">code console</a> and click on the app you've registered (My amazing SMART app). To launch your app through the code console click the "Begin Testing" button. The console will ask if the app you're launching requires a patient in context. Our app requires a patient, so select yes and choose a patient. Please note the millennium username and password, you'll need this credential when prompted. Finally, click launch and the console will redirect to your application.
+
+
+# Run your app against SMART Health IT Sandbox
+
+One of the reasons why SMART on FHIR app is amazing is because of the interoperability factor!  If an EHR follows the SMART and FHIR specifications, your application will work with that EHR's SMART on FHIR implemenmtation.  Let's see if the app that you've built will work with <a href="https://sandbox.smarthealthit.org" target="_blank">SMART Health IT Sandbox</a>.  The following steps will walk you through setting up your app at SMART Health IT Sandbox.
```

Comment:
I was talking about just taking out the word app, and leaving the concept name :)